src/openms/source/ANALYSIS/OPENSWATH/OpenSwathOSWWriter.cpp

```diff
@@ -168,6 +178,8 @@ namespace OpenMS
     {
       score = feature.getMetaValue(score_name).toString();
     }
+    if (score == "nan") score = "NULL";
```

Comment:
is the string "nan" on all platforms (e.g., not NaN)?

Reply:
fixed it by using `toLower()`